src/modules/sih/sih.cpp

```diff
@@ -626,27 +712,33 @@ int Sih::print_status()
 	return 0;
 }
 
+
 int Sih::task_spawn(int argc, char *argv[])
 {
-	Sih *instance = new Sih();
+	_task_id = px4_task_spawn_cmd("sih",
+				      SCHED_DEFAULT,
+				      SCHED_PRIORITY_MAX,
+				      1024,
```

Comment:
That stack seems pretty small.

Reply:
Just tested, does not seem to go beyond 900 bytes of stack

Reply:
You will get stack warnings with 900, there needs to be a margin of 300B.